src/main/java/frc/robot/Subsystems/Drive.java

```diff
@@ -24,14 +24,15 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.DriveConstants;
+import frc.robot.Constants.GlobalConstants;
 import frc.robot.Simulation.DriveSim;
 //#endregion
 public class Drive extends SubsystemBase {
   /** Creates a new Drive. */
 
   //#region declarations
-  public  WPI_TalonFX Left_Front = new WPI_TalonFX(DriveConstants.Left_Front_ID);
-  WPI_TalonFX Right_Front = new WPI_TalonFX(DriveConstants.Right_Front_ID);
+  public static  WPI_TalonFX Left_Front = new WPI_TalonFX(DriveConstants.Left_Front_ID);
```

Comment:
Why are these things (the talons and the methods) marked static? Whatever it is for there has to be a better way to do it?